### DISCUSSION TITLE
Internet House Connection Error

### DISCUSSION THREAD

**Discussion by ArturRod** [rustdesk/rustdesk#2394] [Q&A]
Hello, I have a problem with RustDesk, I don't know if it has to do with the opening of the port in my home. It turns out that if I put data with my mobile I can get to connect to rustdesk and it works perfectly, but if I use the internet at home, be it mine or from anywhere else, this fails. Can someone help me or can know what happens?

**Comment by jabo59** [comment]
I have a nearly similar problem. I cannot connect from my home to a client in Business. But client in Business can connect to client at Home. Own rustdesk server is in place (internet) and all client configured with same key. At home a Fritzbox is in place. When I use the same client (from home) with a direct internet connection (e.g. via smart phone) it works.

Did you solve your problem?

**Comment by ArturRod** [comment]
Hi @jabo59 
I have not been able to solve the problem, I really do not know very well what it is due to.
